### PR TITLE
Submit search by hitting "enter" key and "no results found" edge case

### PR DESCRIPTION
**Pull request by isabelsang**
image

### CODE REVIEW

File: frontend/src/components/dashboard/MuseumGallery.js [hunk 1]
```diff
@@ -80,6 +99,11 @@ export default function MuseumGallery() {
             id="search-textfield"
             label="Search"
             variant="outlined"
+            onKeyUp = {(event) => {
+              if (event.keyCode == 13) {
```
Comment (indhu-gunda) [inline frontend/src/components/dashboard/MuseumGallery.js:103]: change to `===`

File: frontend/src/components/dashboard/MuseumGallery.js [hunk 2]
```diff
@@ -70,7 +71,25 @@ export default function MuseumGallery() {
   useEffect(() => {
     dispatch(fetchMuseumArtworks(museumPage, limit, searchQuery));
   }, [dispatch, museumPage, searchQuery]);
-
+  let paginationNeeded = true;
+  let results;
+  if (artworks.length != 0){
```
Comment (indhu-gunda) [inline frontend/src/components/dashboard/MuseumGallery.js:76]: change to `!==`

File: frontend/src/components/dashboard/MuseumGallery.js [hunk 3]
```diff
@@ -70,7 +71,7 @@ export default function MuseumGallery() {
   useEffect(() => {
     dispatch(fetchMuseumArtworks(museumPage, limit, searchQuery));
   }, [dispatch, museumPage, searchQuery]);
-
+  const noResults = (artworks.length == 0 && museumPage == 1);
```
Comment (BearBurg) [inline frontend/src/components/dashboard/MuseumGallery.js]: use === instead https://google.github.io/styleguide/jsguide.html#features-equality-checks.

File: frontend/src/components/dashboard/MuseumGallery.js [hunk 4]
```diff
@@ -114,15 +115,22 @@ export default function MuseumGallery() {
...
-      <Gallery artworks={artworks}/>
-      <Container className={classes.pagination}>
-        <Pagination
-          count={10}
-          size="large"
-          page={museumPage}
-          onChange={handleChangePage}
-        />
-      </Container>
+      {(artworks.length != 0) ?
```
Comment (BearBurg) [inline frontend/src/components/dashboard/MuseumGallery.js]: use !== instead https://google.github.io/styleguide/jsguide.html#features-equality-checks

**Pull request merged**